app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -582,6 +582,8 @@ const vrb::Vector kAverageHeight(0.0f, 1.7f, 0.0f);
 const vrb::Vector kAverageOculusHeight(0.0f, 1.65f, 0.0f);
 
 struct DeviceDelegateOculusVR::State {
+  const static uint32_t EYE_SURFACE_COUNT = 2;
```

Comment:
nit: I'd use a better name to identify with external, eg. `EXTERNAL_SURFACE_BUFFER_SIZE`